src/JsonModelGenerator/Program.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace JsonModelGenerator
+{
+    class Program
+    {
+        private static readonly IReadOnlyDictionary<string, string> BuiltInTypes = new Dictionary<string, string>(StringComparer.Ordinal)
+        {
+            { "int", "Int" },
+            { "string", "String" },
+            { "Guid", "Guid" },
+            { "DateTime?", "NullableDateTime" },
+            { "bool", "Bool" }
+        };
+
+        public static string GenerateCode(string sourceFilePath, string @namespace)
```

Comment:
This method is quite long. Perhaps break it into multiple methods:
- one to read the source file into a model
- one to generate the output

Reply:
Extracted two methods from this to try to help make it more readable